plugins/find/dialogs/find.js

```diff
@@ -557,6 +557,61 @@
 		}
 
 		var lang = editor.lang.find;
+		function execFindOrReplace( dialog, type ) {
```

Comment:
I would split find and replace to different functions for readability and have dedicated `execFind` / `exedReplace` functions.